app/src/main/java/me/ccrama/redditslide/Activities/MainActivity.java

```diff
@@ -620,8 +621,10 @@ public void onClick(@NonNull MaterialDialog dialog, @NonNull DialogAction which)
                 mEdgeSize.setAccessible(true);
                 int edge = mEdgeSize.getInt(draggerObj);
 
-                mEdgeSize.setInt(draggerObj, edge * 6);
+                mEdgeSize.setInt(draggerObj, edge * DRAWER_EDGE_FACTOR);
             } catch (Exception e) {
+                LogUtil.v("exception while changing side drawer edge" + e);
```

Comment:
Change this to `LogUtil.e(e + ": Exception thrown while changing navdrawer edge size)`, and then delete the two white space lines below this (line 627 and 628). After that, we're more than likely good to merge.